Cortex XDRIR add commands (#26211)
demisto/content: Packs/CortexXDR/Integrations/CortexXDRIR/README.md

                "user_first_name": "dummy",
                "user_last_name": "dummy",
                "user_type": "CSP"
            },
            {
                "groups": [],
                "last_logged_in": null,
                "role_name": "dummy",
                "scope": [],
                "user_email": "dummy@dummy.com",
                "user_first_name": "dummy",
                "user_last_name": "dummy",
                "user_type": "CSP"
            },
          
        ]
    }
}
```

#### Human Readable Output

>### Users
>|First Name|Groups|Last Name|Role|Type|User email|
>|---|---|---|---|---|---|
>| dummy |  | dummy | dummy | CSP | dummy@dummy.com |
>| dummy |  | dummy | dummy | CSP | dummy@dummy.com |

### xdr-list-risky-hosts

***
Retrieve the risk score of a specific host or list of hosts with the highest risk score in the environment along with the reason affecting each score.
Required license: Cortex XDR Pro per Endpoint, Cortex XDR Pro, or Cortex XDR Pro per TB.

#### Base Command

`xdr-list-risky-hosts`

#### Input

| **Argument Name** | **Description** | **Required** |
| --- | --- | --- |
| host_id | Unique ID of a specific host.<br/> | Optional | 

> | limit | Limit the number of hosts that will appear in the list. By default, the limit is 50 hosts.(use limit when you don't asking for specific host). Default is 50. | Optional | 

Reviewer: ```suggestion
| limit | Limit the number of hosts that will appear in the list. (Use limit when no specific host is requested). Default is 50. | Optional | 
```
